### PR TITLE
Cfg improvements

### PR DESCRIPTION
**Pull request by James-Millan**

### CODE REVIEW

File: Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingClassesNode.cs [hunk 1]
```diff
@@ -55,6 +55,11 @@ public OSMBuildingData(List<Vector3> footprint, OSMTags tags)
         MakeRelative();
 		SetHeightAndLevels(tags.height, tags.levels);
 		SetElevation(footprint);
+
+		if (tags.building == "museum" || tags.tourism == "museum")
```
Comment (GeorgeWigley) [inline Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingClassesNode.cs:59]: Could be a cleaner way of doing this?

File: Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingClassesNode.cs [hunk 2]
```diff
@@ -69,7 +74,13 @@ public OSMBuildingData(List<Vector3> footprint, Vector2[][] holes, OSMTags tags)
         this.grammar = Grammars.relations;
         MakeRelative();
         SetHeightAndLevels(tags.height, tags.levels);
+        tags.levels -= 2;
         SetElevation(footprint);
+        
+        if (tags.building == "museum" || tags.tourism == "museum")
+        {
+	        this.grammar = Grammars.museum;
```
Comment (GeorgeWigley) [inline Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingClassesNode.cs:82]: Not sure this is DRY as its a repeat of the clumsy inelegant solution from previous comment

File: Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingClassesNode.cs [hunk 3]
```diff
@@ -109,13 +120,18 @@ private void SetHeightAndLevels(int height, int levels)
...
+            this.buildingLevels = (int)buildingHeight / 4;
         }
         else
         {
             this.buildingLevels = 6;
         }
 
+        // if (this.buildingLevels > this.buildingHeight / 4)
+        // {
+	       //  this.buildingLevels = (int)Math.Floor(this.buildingHeight / 3);
```
Comment (GeorgeWigley) [inline Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingClassesNode.cs:132]: Shouldn't be pushing commented code to dev, please the commit guidelines

File: Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingGameObjectsNode.cs [hunk 4]
```diff
@@ -1,8 +1,10 @@
 ﻿using System;
 using System.Collections.Generic;
+using Unity.VisualScripting;
```
Comment (GeorgeWigley) [inline Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingGameObjectsNode.cs:3]: When is this used?

File: Assets/ProceduralPipeline/Nodes/Buildings/OSMBuildingDataWaysNode.cs [hunk 5]
```diff
@@ -95,6 +95,13 @@ public struct OSMTags
     public int levels;
     public int height;
     public string area;
+    public string amenity;
+    public string tourism;
+    public string leisure;
+    public string shop;
+    public string historic;
+    public string man_made;
+    public string building;
```
Comment (GeorgeWigley) [inline Assets/ProceduralPipeline/Nodes/Buildings/OSMBuildingDataWaysNode.cs:104]: Is this an exhaustive list?

File: Assets/Scripts/WorldGeneration/ContextFreeGrammars/BuildingAssets.cs [hunk 6]
```diff
@@ -40,5 +48,155 @@ public static class BuildingAssets
...
+    
+    public static List<double> getWindowDistribution()
+    {
+        return new List<double>()
+        {
+            0.2,0.02, 0.2, 0.15, 0.01, 0.07, 0.15, 0.1,0.1,
+        };
+    }
+    
+    public static int getMaterialIndex(double probability)
```
Comment (GeorgeWigley) [inline Assets/Scripts/WorldGeneration/ContextFreeGrammars/BuildingAssets.cs:105]: Same again

**Review by GeorgeWigley — APPROVED** [review]
Some minor issues but overall

File: Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingClassesNode.cs (hunk 1 above)
Comment (James-Millan) [reply Assets/ProceduralPipeline/Nodes/Buildings/GenerateOSMBuildingClassesNode.cs:59]: Could you please take another look when it is convenient? Thanks

**Pull request merged**